core/src/hash_table/pair.rs

```diff
@@ -169,7 +158,7 @@ pub mod tests {
         let t = "fooType";
 
         let e1 = Entry::new(t, "bar");
-        let p1 = Pair::new(&chain, &e1);
+        let p1 = chain.create_next_pair(&e1);
 
         assert!(p1.validate());
     }
```

Comment:
Same as with header. This should not be here, but in chain.

Reply:
Fixed